scooter_perception/setup.cfg

```diff
@@ -0,0 +1,4 @@
+[develop]
```

Comment:
Is this used in the CMake? is there a reason it's listed separately? Is this the python install stuff you mentioned?

Reply:
ngl I have no idea what this is but I assume it is part of the ament_python build system which is being used for the perception package